sys/arm64/conf/std.dev

```diff
@@ -95,6 +95,7 @@ device		ehci			# EHCI USB interface (USB 2.0)
 device		xhci			# XHCI USB interface (USB 3.0)
 device		usb			# USB Bus (required)
 device		ukbd			# Keyboard
+device		ums			# USB mouse
```

Comment:
Not needed, as discussed in the upstream review (and I can confirm plugging a USB mouse into CheriBSD builds without this just work)

Reply:
:+1: